src/compat/predicate/isElement.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { describe, expect, it } from 'vitest';
+import { isElement } from './isElement';
+import { args } from '../_internal/args';
+import { falsey } from '../_internal/falsey';
+import { slice } from '../_internal/slice';
+import { symbol } from '../_internal/symbol';
+
+/**
+ * @see https://github.com/lodash/lodash/blob/afcd5bc1e8801867c31a17566e0e0edebb083d0e/test/isElement.spec.js
+ */
+
+describe('isElement', () => {
+  /*
+  it( 'should return `true` for elements', () => {
+    if ( document ) {
+      expect( isElement( body ) ).toBe( true );
+    }
+  } );
+   */
```

Comment:
Unfortunately, this test and the one at the bottom of the file require either access to DOM or at least a DOM mocking library like `happy-dom` or `jsdom`.

Consider using Vitest's [Browser Mode](https://main.vitest.dev/guide/browser/), which without isolation is still really fast.